src/routes/register_startup.js

```diff
@@ -17,6 +17,6 @@ module.exports = (req, res) => {
     if (err) {
       throw err;
     }
-    res.redirect('/')
+    res.redirect('/submitMessage')
```

Comment:
You have a 'return home' button on the submission message view.. and I don't think we need to use setTimeout, it's not a good user experience in my opinion. specially that currently we're not sending them an email to confirm they've submitted so if they miss the  message it might be confusing form them.